ethsigner/commandline/src/main/java/tech/pegasys/ethsigner/EthSignerBaseCommand.java

```diff
@@ -44,6 +48,47 @@
     footer = "EthSigner is licensed under the Apache License 2.0")
 public class EthSignerBaseCommand implements Config {
 
+  static class TlsServerOptions implements TlsOptions {
+
+    @Option(
+        names = "--tls-keystore-file",
+        description =
+            "Path to PKCS#12 format trust store for the JSON-RPC HTTP service client trust.",
+        arity = "1",
+        required = true)
+    private File keystoreFile;
+
+    @Option(
+        names = "--tls-keystore-password-file",
+        description = "Path to a file containing the password required to use the truststore file.",
```

Comment:
truststore or trust store?

Reply:
neither. It should be "key store file" here.